collectors/htcondor/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+package-dir = {""= "src"}
+packages = ["auditor_htcondor_collector"]
+
+[project]
+name = "auditor-htcondor-collector"
+description = "AUDITOR collector for aggregating data from the HTCondor batch system"
+version = "0.0.1"
```

Comment:
This should be

```
version = "0.1.0"
```

because we've decided to use one version number for auditor and all its plugins/collectors. And 0.1.0 is the current version.

Reply:
Sure, this is addressed in [0f93cc2](https://github.com/ALU-Schumacher/AUDITOR/pull/389/commits/0f93cc2a1254b4cf99c29a6758614645e4c188f8).